src/KafkaFlow.SchemaRegistry/KafkaFlow.SchemaRegistry.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Missing PackageId, Description and PackageIconUrl tags

Reply:
Is this project intended to be packaged as a nuget? I think it does only provide internal services

Reply:
Yes, it is.
If we wouldn't publish it on nuget it won't be available to the client applications and the other packages that use it won't be able to be installed 😃

Reply:
I think the assembly will be simply packaged as a direct dependency inside the serializer’s package. It would be rather confusing if it will be published autonomously

Reply:
As this package will be a dependency of the other three packages, it is better to publish it autonomously.  Because we can have a package conflict between different serialization types. Imagine if an application uses the ConfluentProtobuf and ConfluentAvro with different package versions, which version of the SchemaRegistry will be used? We need to let the client choose what version of the package they want to use.

Another situation is where the client application wants to create your own serialization with schema registry support, they can use the SchemaRegistry package only.

Reply:
I understand your position, but consider that the project is currently exposing nothing more than an extension method on the **IClusterConfigurationBuilder**. There is no public API effectivily exposed that's in some form related to the SchemaRegistry.

As you are the maintener of the project I will definetly follow your guidance.